diesel_codegen/src/schema_inference.rs

```diff
@@ -62,7 +62,7 @@ fn column_def_tokens(
     let path_segments = column_type.path
         .into_iter()
         .map(syn::Ident::new)
-        .map(syn::PathSegment::ident)
+        .map(syn::PathSegment::from)
```

Comment:
I added a bunch of `From` impls to syn a while back. I don't think we need the `.map(syn::Ident::new)` here anymore.